### ISSUE TITLE
Border fills node and ignores radius for wasm32

### ISSUE THREAD

**Issue by oscrim** [bevyengine/bevy#14970]
## Bevy version

4648f7b

## Relevant system information
```ignore
AdapterInfo { name: "ANGLE (NVIDIA Corporation, NVIDIA GeForce RTX 2070 SUPER/PCIe/SSE2, OpenGL 4.5.0)", vendor: 4318, device: 0, device_type: Other, driver: "", driver_info: "WebGL 2.0 (OpenGL ES 3.0 Chromium)", backend: Gl }
```

## What you did

Ran the `borders` example targeting wasm32

## What went wrong

When I run the `borders` example targeting x86 it all renders as it should but when the target is wasm32 its rendered incorrectly.
![image](https://github.com/user-attachments/assets/4623b97e-9f6d-4d83-8ff0-ae25405c80e8)

## Additional information

Initially posted this as a comment in #14955, [link to comment](https://github.com/bevyengine/bevy/issues/14955#issuecomment-2315206272)

Everything renders correctly on my laptop
```ignore
AdapterInfo { name: "ANGLE (AMD, AMD Radeon Vega 8 Graphics (raven LLVM 15.0.7), OpenGL 4.6)", vendor: 4098, device: 0, device_type: Other, driver: "", driver_info: "WebGL 2.0 (OpenGL ES 3.0 Chromium)", backend: Gl }
```

Also tested it on my computer at work
```ignore
AdapterInfo { name: "ANGLE (Intel, Mesa Intel(R) UHD Graphics 770 (ADL-S GT1), OpenGL 4.6)", vendor: 32902, device: 0, device_type: IntegratedGpu, driver: "WebGL 2.0 (OpenGL ES", driver_info: "3.0 Chromium)", backend: Gl }
```

One thing that stands out when comparing the adapter info is that on the computers where its rendered correctly its using OpenGL 4.6 while the other computer uses OpenGL 4.5.0

**Comment by ickshonpe** [comment]
Does https://github.com/bevyengine/bevy/pull/15163 help at all with this?

**Comment by oscrim** [comment]
> Does #15163 help at all with this?

Not at all unfortunately, but it seems to be even more broken now on bevy main.

### Your branch 
![image](https://github.com/user-attachments/assets/58764b95-f302-45dd-8d17-c394234b3d35)

### Bevy main (8bfe635)
![image](https://github.com/user-attachments/assets/12879a16-99dd-4dbb-99b1-b56ad48cd648)

**Comment by oscrim** [comment]
Also tried it with the commit I used originally in the issue and it looks like it did before. Im going to try to bisect it to see what commit made it worse

**Comment by oscrim** [comment]
[a0f5ea0](https://github.com/bevyengine/bevy/commit/a0f5ea0d3621a14bdc06d1e3bf1a8d9de90d409b) is the commit that broke it further

**Comment by ickshonpe** [comment]
Yeah that makes sense, that commit changed outlines to use the shader to draw the outlines with border radius too.

I could add a mode to the shader which would instruct it to ignore border radius and always draw square borders. It's not ideal but at least apps would be usable again.

**Comment by oscrim** [comment]
Yeah, that useable apps are preferable :sweat_smile:  Let me know if you need me to do any more testing

**Comment by oscrim** [comment]
This is still an issue, below is a screenshot from the Borders example on the bevy website.

![Image](https://github.com/user-attachments/assets/70bfbce7-f64e-4320-9850-491fb282860f)

Ive now also seen it on a colleagues computer, he also has an Nvidia GPU and is on Linux so the problem seems to be specifically with WASM on Linux machines with an Nvidia GPU.

**Comment by Jaewookim08** [comment]
I'm having the same issue. I'm seeing the same screen as the image above, running on a Linux machine with an Nvidia GPU, only in Wasm.

**Comment by hexroll** [comment]
I think it's the anti-aliased version of the sdfs:
https://github.com/bevyengine/bevy/blob/1030a99b8e2680a7e696d6433b79f5671768231c/crates/bevy_ui/src/render/ui.wgsl#L166

Without anti-aliasing I got the rounded-borders to work.
